core-plugin/build.gradle

```diff
@@ -1,64 +1,53 @@
-apply from: '../scripts/javac2.gradle'
-def ideaHome = '../idea-IC/'
-def pluginName = 'google-cloud-tools-core'
+plugins {
+    id "org.jetbrains.intellij" version "0.0.29"
+}
+
+// **replace** default sourceSets with idea layout
```

Comment:
Do you think we can standardize around the gradle layout eventually? Or is that a bad idea?